.github/dependabot.yaml

```diff
@@ -0,0 +1,29 @@
+---
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: "chore(deps): github-actions"
+      include: "scope"
+    labels:
+      - "github-actions"
+      - "dependencies"
+  - package-ecosystem: docker
```

Comment:
If docs it can also update kubernetes manifest files but I'm not sure it can do that for those deployed by kustomize.

Reply:
It might check automatically or we'll need to add some extra path.
We'll check on that later on or on another repository set at a higher level than the runner-manager

Reply:
ok